zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -86,7 +86,7 @@
             <i ng-if="!navbar.connected" class="fa fa-circle server-disconnected"
                uib-tooltip="WebSocket Disconnected" tooltip-placement="bottom" style="margin-top: 7px; vertical-align: top"></i>
             <button ng-if="ticket" class="nav-btn dropdown-toggle" type="button" data-toggle="dropdown" style="margin:11px 5px 0 0; padding-left: 0px;">
-              <span class="username">{{ticket.principal}}</span>
+              <span class="username">{{ticket.screenUsername}}</span>
```

Comment:
reference https://github.com/bujiio/buji-pac4j/blob/01cf4f8f0c5e7cbe76d4697d7d28873d6d542de4/README.md#5-get-the-user-profile